kotlin-runtime/ftl-runtime/src/main/kotlin/xyz/block/ftl/schemaextractor/ExtractSchemaRule.kt

```diff
@@ -558,7 +559,15 @@ class SchemaExtractor(
     }
 
     private fun KtDeclaration.comments(): List<String> {
-      return this.docComment?.text?.trim()?.let { listOf(it) } ?: emptyList()
+      return this.docComment?.text?.trim()?.let { text ->
+        text.removePrefix("/*")
+          .removeSuffix("*/")
+          .lines()
+          .map { it.trim().removePrefix("*").trim() }
+          .filter { it.isNotEmpty() }
```

Comment:
I don't think we want to remove empty lines in the comments do we?

Reply:
ah the thinking here was to filter out the lines that become empty after removing the javadoc start/end markers `/**` and `*/`- but good point, we don't want to lose intentional line spacing. I've updated to only filter out the lines that contain start/end markers with nothing else